tools/project_manager/diff.php

```diff
@@ -172,6 +172,42 @@ function copyToClip(textstring) {
     echo $navigation_text;
 }
 
+// Load an associative array for which pages in the project have diffs.
```

Comment:
Since we're trying to get better about this could we add
```
/**
  * @return array<string, array<string, mixed>> 
  * Return an associative array indexed by page image name, where each page has an array of attributes.
*/
function load_page_diff_array(Project $project, Round $L_round, Round $R_round, bool $include_no_format = false): array
```